src/main/java/seedu/address/storage/JsonSerializablePropertyBook.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.PropertyBook;
+import seedu.address.model.ReadOnlyPropertyBook;
+import seedu.address.model.property.Property;
+
+/**
+ * An Immutable PropertyBook that is serializable to JSON format.
+ */
+@JsonRootName(value = "propertybook")
+class JsonSerializablePropertyBook {
+
+    public static final String MESSAGE_DUPLICATE_PROPERTY = "Property list contains duplicate properties.";
+
+    private final List<JsonAdaptedProperty> properties = new ArrayList<>();
+
+    /**
+     * Constructs a {@code JsonSerializablePropertyBook} with the given appointments.
```

Comment:
Should update the comment to `properties`

Reply:
fixed

Reply:
i think he is referring to line 27, the last word before the full stop